miden/src/cli/run.rs

```diff
@@ -106,16 +110,19 @@ fn run_program(params: &RunCmd) -> Result<(ExecutionTrace, [u8; 32]), Report> {
     let libraries = Libraries::new(&params.library_paths)?;
 
     // load program from file and compile
-    let program =
-        ProgramFile::read(&params.assembly_file)?.compile(&Debug::Off, &libraries.libraries)?;
+    let program = ProgramFile::read(&params.assembly_file)?
+        .compile(&Debug::from(params.debug), &libraries.libraries)?;
```

Comment:
Could we do this as:
```rust
let program = ProgramFile::read(&params.assembly_file)?
    .compile(params.debug.into(), &libraries.libraries)?;
```
We'll need to change the signature of `ProgramFile::compile()` for this - but I think that's fine.

Reply:
Yes, this works. Would this need to be mentioned in the change log?

Reply:
I think this is not a part of the external interface - so, I wouldn't mention it.